config.example.json

```diff
@@ -1,5 +1,56 @@
 {
     "username": "youremailaddress@gmail.com",
     "password": "yourpassword",
-    "min_value": 100
+    "min_value": 100,
```

Comment:
Is there a higher level logger available that just contains those defaults and lets me literally just import it and start log()'ing? Configuring the logger looks so big!